packages/query-core/src/query.ts

```diff
@@ -167,6 +169,7 @@ export class Query<
   #initialState: QueryState<TData, TError>
   #revertState?: QueryState<TData, TError>
   #cache: QueryCache
+  #client: QueryClient
```

Comment:
Would be nice to get a getter for QueryClient from the instance of Query. 
I missed that before for experimental persister.

Reply:
you probably don’t need that now that it’s passed to the `context`, which the persister also has access to.